docs/proposals/edgepodgetandrestart.md

```diff
@@ -0,0 +1,142 @@
+---
+title: When the edge node goes offline, add pod restart and status query functions for the edge node
+authors:
+- "@luomengY"
+  approvers:
+  creation-date: 2024-04-15
+  last-updated: 2024-04-15
+  status: implementable
+---
+
+# When the edge node goes offline, add pod restart and status query functions for the edge node 
+
+
+## Motivation 
+
+When the edge node is offline, it is not possible to query the status of the edge node pod and restart the pod of the edge node through kubectl in the cloud. This feature provides support for querying and restarting the pod status of the edge node.
```

Comment:
> The scene in the last sentence is not very clear. Maybe it can be changed to "This feature provides support for querying and restarting the pod status **from edge** with **offline edge node** " . You can check whether this statement is correct

In fact, pods can be queried and restarted both online and offline.